<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>96</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Account Addition</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
background:white;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>6</number>
   </property>
   <item row="1" column="4">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Character: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QComboBox" name="comboBoxChannel"/>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Channel: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboBoxWorld"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Username: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="labelUsername"/>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Password: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="labelPassword"/>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>2nd Password: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QLineEdit" name="labelSecondaryPassword"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>World: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QComboBox" name="comboBoxCharacter"/>
   </item>
   <item row="2" column="0" colspan="6">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="6">
    <widget class="QDialogButtonBox" name="dialogButtonBoxAddAccount">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>comboBoxWorld</sender>
   <signal>activated(int)</signal>
   <receiver>Dialog</receiver>
   <slot>ComboBox_Activated()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>-163</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialogButtonBoxAddAccount</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>Dialog</receiver>
   <slot>DialogButtonBox_Clicked(QAbstractButton*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>392</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ComboBox_Activated()</slot>
  <slot>DialogButtonBox_Clicked(QAbstractButton*)</slot>
 </slots>
</ui>
